src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/config/BackupProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+
+@Component
+@Data
+@ConfigurationProperties("backup")
+public class BackupProperties {
+    private String gcsBackupBucket;
```

Comment:
If you have the `spring-boot-starter-validation` dependency, you could also add `@NotBlank` here to assert the configuration value at startup.